### ISSUE TITLE
question about the Model-MetaP-Mbv2-140M accuracy

### ISSUE THREAD

**Issue by diligent321** [liuzechun/MetaPruning#33]
after load the pretrain model, when i start the evaluate process, the top1 accuracy of first batch in training dataset is only 46.09%, so why the train accuracy is so slow? 

the steps are as follows,
Firstly, i download the  Model-MetaP-Mbv2-140M directory, there are two files, checkpoint.pth.tar and mobilenet_v2.py.
Then, i copy mobilenet_v2.py to evaluating dir, checkpoint.pth.tar to evaluating/models, and run the run.sh script in evaluating dir, the log is in picture,

![image](https://user-images.githubusercontent.com/31379237/82139957-8097c800-985e-11ea-95e6-0c9c13bfe3c3.png)

Besides, when i comment the train part in evaluate.py, the accuracy of eval dataset is similar.

**Comment by liuzechun** [comment]
Hi, Thanks for your interest in my work. 

Actually the ImageNet data I used has a little different normalization range with the pytorch imagenet data. You can check this data augmentation in pytorch https://github.com/megvii-model/MABN/blob/master/cls/utils.py My college implemented that and it can reproduce the test accuracy. 

Sorry for the confusion and I will modify the data preprocessing and augmentation code in this repo very soon.

**Comment by diligent321** [comment]
Thanks for your kindly reply,

after changing the code according to your suggestion, the pretrain model can reproduce the test accuracy.

**Comment by he-y** [comment]
@diligent321 Hi, can you provide some information about how to implement the data augmentation?
Thank you!
